configuration/csv-to-docs.md

```diff
@@ -1,32 +1,276 @@
+#### Get Value From Another Doc
 
-Run command `medic-conf fetch-csvs-from-google-drive`. This will download the CSV files in the json document created above and place them into a folder named `csv`.
+Similar to including another doc, it is also possible to get the value of a specific field in another doc. For instance, if `parent:GET _id OF place WHERE reference_id=COL_VAL` were used in the example above, the `parent` field's value would have been set to the `_id` of the refered to doc instead of including the whole doc. Note that `_id` is a generated value included in all generated docs.
 
-Running `medic-conf csv-to-docs` will convert the CSV into json docs for the webapp. They are placed in a `json_docs` folder
+| reference_id:excluded | parent:GET _id OF place WHERE reference_id=COL_VAL | is_name_generated | name | reported_date:timestamp |
+| --------------------- | -------------------------------------------------- | ----------------- | ---- | ----------------------- |
+| health_center_1       | district_1                                         | false             | HC1  | 1544031155715           |
+| health_center_2       | district_2                                         | false             | HC2  | 1544031155715           |
+| health_center_3       | district_3                                         | false             | HC3  | 1544031155715           |
+
+The resulting doc structure would be:
+
+```
+{
+  "type": "health_center",
+  //Parent property with the _id from district_1 as the value.
+  "parent": "0c31056a-3a80-54dd-b136-46145d451a66",
+  "is_name_generated": "false",
+  "name": "HC3",
+  "external_id": "",
+  "notes": "",
+  "geolocation": "",
+  "reported_date": 1544031155715,
+  "_id": "45293356-353c-5eb1-9a41-baa3427b4f69"
+}
+```
 
-Lastly run `medic-conf --local upload-docs` to upload the converted docs into your local instance. `--local` can be replaced with an instance or URL. See [medic-conf](https://github.com/medic/medic-conf) for detailed instructions.
+## Creating CSV files for Users
 
-### Google Drive Authentication
 
-Medic-conf leverages google authentication to access google drive. You will need to create a client_secrets file named `.gdrive.secrets.json` and place it in the same directory where you want the CSVs to be fetched. [Token Creation](https://developers.google.com/identity/protocols/OAuth2InstalledApp)
+### Creating a new user with a new place
 
-Create the `.gdrive.secrets.json` file by downloading the `client_secrets.json` from google. You will need a CLIENT_ID, CLIENT_SECRET and REDIRECT_URL. You can find these pieces of information by going to the Developer Console, clicking your project --> APIs & auth --> credentials --> Download JSON. This will download the credentials but will need modified to be in this structure. 
+To create new users associated to new place and a new contact. Provide values for contact.name, place.name, and place.parent(can be existing place)
```

Comment:
```suggestion
To create new users associated to a new place and a new contact, provide values for contact.name, place.name, and place.parent (can be an existing place)
```

Reply:
Thanks for finding this mistake. I couldn't accept your suggestion directly anymore, but think it's covered by other corrections.